LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Norse
+Copyright (c) 2021 PMMon
```

Comment:
This has to be adjusted in some way, we should probably add "the AESTREAM authors" there and have an AUTHORS file. But notice how you are actually in the LICENSE *not* allowed to change the License. Now Norse is not a legal entity. The code I wrote either is owned by University of Heidelberg or myself and strictly speaking any new contributor has to either transfer their rights implicitly or explicitly to the library.

Reply:
I changed it to "the AEStream authors" in the LICENSE file and added a AUTHORS file in alphabetical order. It's still on the MIT Licence. Is that ok?

Reply:
Sure